LoopKit/UserAlert.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  UserAlert.swift
```

Comment:
We have other alerts that will be shown to the user. This work pertains to device alerts, so maybe naming this "DeviceAlert" would help describe and scope this.